spec/lib/repo/git_revision_spec.rb

```diff
@@ -12,7 +12,15 @@
 
       # Make sure repo exists, if not make it
       repo = ga_repo.config.get_repo('test_repo_workdir')
+
       if repo.nil?
+        Repository.conf =
+            {
```

Comment:
Indent the first parameter one step more than the start of the previous line.<br>Indent the first line of the right-hand-side of a multi-line assignment.